haas/api.py

```diff
@@ -534,6 +534,17 @@ def headnode_detach_network(headnode, hnic):
                             # Network Code #
                             ################
 
+@rest_call('GET', '/networks')
+def list_networks():
+    """List all networks.
+    Returns a JSON array of strings representing a list of projects.
+    Example:  '["network1", "network2", "network3"]'
+    """
+    db = model.Session()
+    networks = db.query(model.Network).all()
+    network_labels = [n.label for n in networks]
+    network_ids = [n.network_id for n in networks]
+    return json.dumps([network_labels, network_ids], sort_keys=True)
```

Comment:
This will output two parallel lists. I think it would be better to return a list of the JSON blobs returned in show_network().

If implementing the show_network() logic, it would be nice if we could rename "access" to "project", and if there is a project to output it but if not (ie- it's NULL) to output something like "PUBLIC".